utils.py

```diff
@@ -407,3 +407,14 @@ def sortBy(self):
             case "Relevent":
                 sortBy = "sortBy=R"                
         return sortBy
+
+
+def cleanJobMetadata(text: str, separators=['·', '(', '-', '|']) -> str:
```

Comment:
```suggestion
def getFirstStringBeforeSeparators(text: str, separators=['·', '(', '-', '|']) -> str:
```

Reply:
I renamed the function name to your suggested one.